pkg/apis/autoscaling/v1alpha1/federatedhpa_types.go

```diff
@@ -41,17 +41,16 @@ type FederatedHPASpec struct {
 	// +required
 	ScaleTargetRef autoscalingv2.CrossVersionObjectReference `json:"scaleTargetRef"`
 
-	// minReplicas is the lower limit for the number of replicas to which the autoscaler
-	// can scale down.  It defaults to 1 pod.  minReplicas is allowed to be 0 if the
-	// alpha feature gate HPAScaleToZero is enabled and at least one Object or External
-	// metric is configured.  Scaling is active as long as at least one metric value is
-	// available.
+	// MinReplicas is the lower limit for the number of replicas to which the
+	// autoscaler can scale down.
+	// It defaults to 1 pod.
 	// +optional
-	MinReplicas *int32 `json:"minReplicas,omitempty" protobuf:"varint,2,opt,name=minReplicas"`
+	MinReplicas *int32 `json:"minReplicas,omitempty"`
```

Comment:
Do we need `// +kubebuilder:default=1`?

Reply:
This PR focuses on cleanup things. 
If you want to add more validations, please send another PR.

> Do we need // +kubebuilder:default=1?

I don't think so. We handled the default value at webhook. 
https://github.com/karmada-io/karmada/blob/master/pkg/util/lifted/federatedhpa.go#L61-L63